stimuli-ci-generation/cloudbuild.yaml

```diff
@@ -0,0 +1,59 @@
+steps:
```

Comment:
Could you please advise which one of these was actually used and which one was for testing / not relevant? Some comments here would be awesome if you still happen to remember 😄

Reply:
I never used this to deploy. I used the steps in DEPLOY_NOTES.md  to build and push the new container, and then I just restarted the cloud run service.

I'm attaching the .yaml file that is provided by google (that reflects the current running service). LMK if that helps or 

[cloud.yaml.txt](https://github.com/neu-self-image-experiments/sie-backend/files/6018422/cloud.yaml.txt)